README.md

```diff
@@ -113,6 +113,36 @@ _Note: This will only work if you've created your own bot account._
     ```
 - This concept will work for anything in JavaScript. Check out [JavaScript methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Methods_Index) to see what else is possible!     
 
+### Have the Bot interact with an API
+- This code supports Javascript Promises
+- To test this out, here's some code you can start with. It uses [the free OpenWeather API](https://openweathermap.org/current).
+```
+{
+  regex: /\/weather/gim,
+  func: (input) => {
+    return new Promise((resolve, reject) => {
+          request({
+            // note: this API key is public and free. Too many calls will get it blocked.
+            uri: 'http://api.openweathermap.org/data/2.5/weather?q=London,uk&appid=bd5e378503939ddaee76f12ad7a97608',
+            method: 'GET'
+          }, (err, response, body) => {
+            if(err) { reject(err); }
+            else {
+              const jsonBody = JSON.parse(body);
+              resolve(jsonBody.weather[0].description);
+            }
+          });
+        });
+  }
+
+```
+- You can see that the functionality is low level. Make this more fun by changing `London,uk` or by changing which output is shown from the API response. Remember, JavaScript is supported, and you can get messages from the chat users. :wink:
+- Want to do more? Like add giphy support, or other APIs? Start by asking yourself (and google, if necessary) these questions:
+  - What's the API I want to use, and how do I call it?
+  - How can I use more advanced work with regexes, like groups, or string manipulation? How would I _want_ to?
```

Comment:
@OriR What about "How can I use regex to parse the message a user sent as data and respond differently to each message? Something like "/calculate 2+2" and the bot would answer "4". "?